backend/program/scrapers/common.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
```

Comment:
For the logs to be how they are? I fixed that. Looks like _scrape_item has to stay within the scraper modules for this to work. Otherwise, gotta edit the logger module, which I didnt want to do